src/engine/enginebuffer.cpp

```diff
@@ -232,6 +235,12 @@ EngineBuffer::EngineBuffer(const QString& group,
     m_pCueControl = new CueControl(group, pConfig);
     addControl(m_pCueControl);
 
+    for (int i = 0; i < 8; ++i) {
+        auto control = new MacroControl(group, pConfig, i + 1);
```

Comment:
Memory leak if you don't delete these in the destructor

Reply:
They are deleted, together with all other `m_engineControls` :)